pack/tcu/bbt_encounter.json

```diff
@@ -0,0 +1,404 @@
+		"code": "05328",
+		"doom": 8,
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "...that leering, tittering shantak coursed on impetuous and relentless, flapping its great slippery wings in malignant joy, and headed for those unhallowed pits whither no dreams reach... \n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
+		"illustrator": "Lenka Šimečková",
+		"name": "The Final Countdown",
+		"pack_code": "bbt",
+		"position": 328,
+		"quantity": 1,
+		"stage": 3,
+		"text": "Each location is connected to each location adjacent to it.\nDo not remove doom from enemies when this agenda advances.",
+		"type_code": "agenda"
+	},
+	{
+		"back_flavor": "Your steed carries you through the entryway of the nightmarish palace, and you are surprised to find the halls inside to be as vast and cavernous as the empty space surrounding it. It is as though the doorway was but the threshold of another unvierse, as desolate and boundless as the last.",
+		"back_name": "Palace of the Old Ones",
+		"back_text": "Remove Cosmic Ingress from the game. Shuffle each location in play other than Hideous Palace into the Cosmos. Each investigator at those locations is left behind and is killed. Each enemy at those locations is shuffled into the top 5 cards of the encounter deck.\nTake the set-aside Court of the Great Old Ones and the top 2 cards of the Cosmos, and shuffle them so you cannot tell which is which. Then, put them into play along with empty space, as depicted in \"Location Placement for Act 2\" in the Campaign Guide.\nIn player order, each investigator draws the top card of the encounter deck.",
+		"clues": 1,
+		"code": "05329",
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "From the horizonless breach in the gaping maw of the sky, you have traveled an impossible distance, and yet you are still many leagues from the nighted throne where Azathoth stirs.",
+		"illustrator": "Reiko Murakami",
+		"name": "The Cosmos Beckons",
+		"pack_code": "bbt",
+		"position": 329,
+		"quantity": 1,
+		"stage": 1,
+		"text": "[action] Spend X clues: Draw the top X cards of the Cosmos. Choose one to put into play using its \"<b>Cosmos</b> -\" instructions, and move to it. Shuffle the rest back into the Cosmos.\n<b>Objective</b> - Only investigators at Hideous Palace may spend the requisite number of clues, as a group, to advance. (<i>Beware - any investigator who is not there will be killed.</i>)",
```

Comment:
Please can you move the brackets outside the italics tags for the Beware clause?